cmd/kops/delete_cluster.go

```diff
@@ -42,6 +43,8 @@ func init() {
 	cmd.Flags().BoolVar(&deleteCluster.External, "external", false, "Delete an external cluster")
 
 	cmd.Flags().StringVar(&deleteCluster.Region, "region", "", "region")
+
+	cmd.Flags().StringVarP(&deleteCluster.Manifest, "manifest", "f", "", "Optional .yml cluster manifest")
```

Comment:
Should we store the command messages in one file? I am a huge fan of duplicated strings ... not.  This is probably another PR ...